### PR TITLE
fix(Hide): use native tailwind breakpoint range classes

### PR DESCRIPTION
**Pull request by mainframev**
# This Pull Request meets the following criteria:

Fixes reported issue [FEPLT-1819](https://kiwicom.atlassian.net/browse/FEPLT-1819)

[FEPLT-1819]: https://kiwicom.atlassian.net/browse/FEPLT-1819?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
 Storybook: https://orbit-mainframev-fix-hide-and-screens-issue.surge.sh

### CODE REVIEW

File: packages/orbit-tailwind-preset/src/presets/components/index.ts [hunk 1]
```diff
@@ -88,28 +86,6 @@ const cfg = (options?: Options): Config => {
     },
     theme: {
       extend: {
-        screens: {
-          "sm-mm": { max: px(tokens.breakpointMediumMobile - 1) },
-          "sm-lm": {
-            max: px(tokens.breakpointLargeMobile - 1),
-          },
```
Comment (DSil) [inline packages/orbit-tailwind-preset/src/presets/components/index.ts:95]: This is being used in ModalFooter.

File: packages/orbit-components/src/deprecated/Desktop/__tests__/index.test.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ describe("Desktop", () => {
     render(<Desktop>kek</Desktop>);
     expect(screen.getByText("kek")).toMatchInlineSnapshot(`
       <div
-        class="inline-block sm-mm:hidden mm-lm:hidden lm-tb:hidden tb-de:hidden"
+        class="inline-block sm:max-lm:hidden mm:max-lm:hidden lm:max-tb:hidden tb:max-de:hidden"
```
Comment (DSil) [inline packages/orbit-components/src/deprecated/Desktop/__tests__/index.test.tsx:11]: If we had `sm-mm` why did it change to `sm:max-lm`?

File: packages/orbit-components/src/Hide/index.tsx [hunk 3]
```diff
@@ -8,11 +8,11 @@ import type { Props } from "./types";
 const Hide = ({ on = [], block, children }: Props) => (
   <div
     className={cx(block ? "block" : "inline-block", {
-      "sm-mm:hidden": on.includes("smallMobile"),
-      "mm-lm:hidden": on.includes("mediumMobile"),
-      "lm-tb:hidden": on.includes("largeMobile"),
-      "tb-de:hidden": on.includes("tablet"),
-      "de-ld:hidden": on.includes("desktop"),
+      "sm:max-lm:hidden": on.includes("smallMobile"),
```
Comment (DSil) [inline packages/orbit-components/src/Hide/index.tsx:11]: Shouldn't it be?

```suggestion
      "sm:max-mm:hidden": on.includes("smallMobile"),
```

File: packages/orbit-components/src/Hide/__tests__/index.test.tsx [hunk 4]
```diff
@@ -12,7 +12,7 @@ describe("Hide", () => {
     );
     expect(container.childNodes[1]).toMatchInlineSnapshot(`
       <div
-        class="inline-block sm-mm:hidden lm-tb:hidden ld:hidden"
+        class="inline-block sm:max-lm:hidden lm:max-tb:hidden ld:hidden"
```
Comment (DSil) [inline packages/orbit-components/src/Hide/__tests__/index.test.tsx:15]: Same question as above

**Review by DSil — CHANGES_REQUESTED** [review]

File: packages/orbit-tailwind-preset/src/presets/components/index.ts (hunk 1 above)
Comment (mainframev) [reply packages/orbit-tailwind-preset/src/presets/components/index.ts:95]: good catch, thanks

File: packages/orbit-components/src/deprecated/Desktop/__tests__/index.test.tsx (hunk 2 above)
Comment (mainframev) [reply packages/orbit-components/src/deprecated/Desktop/__tests__/index.test.tsx:11]: it's inside Hide, will fix

**Review by DSil — COMMENTED** [review]
LGTM. Organize commits and rebase and I'll approve 👍

**Review by DSil — APPROVED** [review]

**Pull request merged**